products/compute/terraform.yaml

```diff
@@ -15,21 +15,42 @@
 overrides: !ruby/object:Provider::ResourceOverrides
   Address: !ruby/object:Provider::Terraform::ResourceOverride
     exclude: true
+  BackendBucket: !ruby/object:Provider::Terraform::ResourceOverride
+    id_format: "{{name}}"
```

Comment:
Make "{{name}}" the default value for the id_format field.

Reply:
I <3 ResourceOverride. No need to scatter that logic everywhere ;) Done